packages/data/src/world/oot/overworld.yml

```
@@ -982,11 +982,12 @@
     "Lost Woods Deep": "is_child || can_play_saria || trick_mido || hookshot_anywhere || climb_anywhere"
     "Lost Woods Generic Grotto": "has_explosives_or_hammer"
     "Goron City Shortcut": "true"
-    "Zora River": "can_dive_small || hookshot_anywhere"
-    "Lost Woods Rupee Arrow": "is_child && (can_dive_small || hookshot_anywhere || can_boomerang)"
+    "Zora River": "can_dive_small || hookshot_anywhere && can_swim"
```

can_dive_small can just be edited to include can_swim with the scale req. I'll handle that part.
```suggestion
    "Zora River": "can_dive_small || hookshot_anywhere"
```